starboard/android/apk/app/src/main/java/dev/cobalt/media/AudioTrackBridge.java

```diff
@@ -165,15 +166,38 @@ public AudioTrackBridge(
       audioTrackBufferSize /= 2;
     }
     if (isAudioTrackValid()) {
-      audioTrackBufferSize = audioTrack.getBufferSizeInFrames() * channelCount * getBytesPerSample(sampleType);
+      // If the AudioTrack encoding indicates compressed data,
+      // e.g. AudioFormat.ENCODING_AC3, then the frame count returned
+      // is the size of the AudioTrack buffer in bytes.
```

Comment:
Add: "In such cases `audioTrackBufferSize` doesn't have to be multiplied with bytes per sample and channel count below".

Reply:
Fixed.